lib/ViewModels/NowViewingTabViewModel.js

```diff
@@ -57,8 +58,13 @@ var NowViewingTabViewModel = function(options) {
             if (!oneIsOpen && nowViewingItems.length > 0) {
                 nowViewingItems[0].isLegendVisible = true;
             }
+
+            window.setTimeout(function(){
```

Comment:
@kring  I think this is needed, otherwise when you switch from data catalogue tab to now viewing tab using the keyboard, it breaks the transition :)